notebook/nbextensions.py

```diff
@@ -831,7 +884,60 @@ class DisableNBExtensionApp(ToggleNBExtensionApp):
     Usage
         jupyter nbextension disable [--system|--sys-prefix]
     """
-    _toggle_value = None
+    version = __version__
+    flags = {
+        "py" : ({"DisableNBExtensionApp" : {"python" : True}},
+            "Disable an extension from a Python package name"),
+        "user": ({"DisableNBExtensionApp" : {"user": True}},
+            "Explicitly disable in user config"),
+        "sys-prefix": ({"DisableNBExtensionApp": {"sys_prefix": True}},
+            "Explicitly disable in sys.prefix config")
+    }
+    flags['python'] = flags['py']
+    aliases = {'section': 'DisableNBExtensionApp.section'}
+
+    user = Bool(False, config=True, help="Explicitly disable in user config")
+    sys_prefix = Bool(False, config=True,
+                      help="Explicitly disable in sys.prefix config")
+
+    python = Bool(False, config=True, help="Install from a Python package")
+
+    section = Unicode('notebook', config=True,
+                      help="""Which config section to disable the extension in."""
+                      )
+
+    def start(self):
```

Comment:
Could we get away with only overriding start, instead of losing the ToggleNBExtensionApp inheritance?